### DISCUSSION TITLE
Cached connection details?

### DISCUSSION THREAD

**Discussion by Vichman67** [dbeaver/dbeaver#23474] [Q&A]
Hello,

I'm using DBeaver Enterprise Edition release 24.0.0.202403110838 under Windows 10 Education. My problem deals with connections to MySQL servers. As you might know, when creating a MySQL database user you have to specify the database user name plus the source IP originating the connection, e.g. datacv@10.157.12.23.

I've recently changed the IP address of the computer running DBeaver client (new IP address - 10.157.200.15). Since then, I can't connect to MySQL servers. My MySQL DBA has properly changed the usernames I use for connections so they reflect my new IP address. In fact, I can connect to those MySQL servers from my PC using other DB clients, so I'm sure there is  problem with DBeaver.

When trying to connect from DBeaver, the error obtained is kind of:
`Access denied for user 'datacv'@'10.157.200.15' to database 'moodle'`
But it reflects my new IP (10.157.200.15), and as I said using that new IP and DB username I can establish successful connections from the same PC via other DB clients.

I've tried to create a new MySQL connection but the problem persists.

So I'm wondering if DBeaver has a kind of connections cache and if it is possible to delete / refresh it.

Any idea will be welcome.

TIA.

**Comment by LonwoLonwo** [comment]
Hello @Vichman67 

No, DBeaver doesn't have a connection cache.
But did you try to restart DBeaver? 
And did you check your connection settings - host, user name?

**Comment by Vichman67** [comment]
Finally it was due to a strange behaviour on the MySQL server side. Our DBA didn't properly changed the MySQL usernames - he had to grant them some other privileges. And the fact that I could connect from other DB clients was not accurate - the connection proceeded but after that there was no way to dive into the DB structure.

So DBeaver is working great, as usual, and this discussions can be closed. Thanks a lot!

**Reply by LonwoLonwo** [reply]
Thanks for the update!
